### PR TITLE
Added 'add to google calenda' button

### PR DESCRIPTION
**Pull request by gijsbel**
Should still create translation for the text of the button.

### CODE REVIEW

File: module/Activity/view/activity/activity/view.phtml [hunk 1]
```diff
@@ -74,6 +74,27 @@ $this->headTitle($this->translate('Activities')); ?>
...
                                         </a>
                                     <?php endif ?>
+									
+									<?php
+										$pre = "https://calendar.google.com/calendar/render?action=TEMPLATE";
+										
+										$text = "&text=".$activity->getName();
+										
+										$begin = $activity->getBeginTime();
+										$begin = $begin->format('Ymd')."T".$begin->format('His');
```
Comment (jszanto) [inline module/Activity/view/activity/activity/view.phtml]: Why not just `$activity->getBeginTime()->format('Y-m-d\TH:i:s');`?

File: module/Activity/view/activity/activity/view.phtml [hunk 2]
```diff
@@ -74,6 +74,27 @@ $this->headTitle($this->translate('Activities')); ?>
...
+										
+										$end = $activity->getEndTime();
+										$end = $end->format('Ymd')."T".$end->format('His');
+										
+										$dates = "&dates=".$begin."/".$end;
+										
+										$location = "&location=".$activity->getLocation();
+										
+										$link = $pre.$text.$dates.$location;
```
Comment (jszanto) [inline module/Activity/view/activity/activity/view.phtml]: All this stuff should be urlencoded for this to work.

Comment (jszanto) [reply module/Activity/view/activity/activity/view.phtml]: Just a suggestion to make this a lot easier
```
$link = 'https://calendar.google.com/calendar/render?action=TEMPLATE?';
$link .= http_build_query([
    'text' =>  $activity->getName(),
    'dates' => $begin . '/' . $end,
    '$location = $activity->getLocation(),
]);

File: module/Activity/view/activity/activity/view.phtml [hunk 3]
```diff
@@ -74,6 +74,27 @@ $this->headTitle($this->translate('Activities')); ?>
                                             />
                                         </a>
                                     <?php endif ?>
+									
+									<?php
```
Comment (jszanto) [inline module/Activity/view/activity/activity/view.phtml]: Please indent with multiples of 4 spaces

**Comment by gijsbel** [comment]
Thanks for the feedback. Will revise the code.

**Comment by gijsbel** [comment]
Done 💯

File: module/Activity/view/activity/activity/view.phtml [hunk 4]
```diff
@@ -74,6 +74,20 @@ $this->headTitle($this->translate('Activities')); ?>
                                             />
                                         </a>
                                     <?php endif ?>
+                                    
+                                    <?php
+                                        $begin = $activity->getBeginTime()->format('Ymd\THis');
+                                        $end = $activity->getBeginTime()->format('Ymd\THis');
```
Comment (Kevinjil) [inline module/Activity/view/activity/activity/view.phtml]: A call to `getBeginTime()` is made while it should have been `getEndTime()`.

File: module/Activity/view/activity/activity/view.phtml [hunk 5]
```diff
@@ -74,6 +74,20 @@ $this->headTitle($this->translate('Activities')); ?>
...
+                                        $link = 'https://calendar.google.com/calendar/render?action=TEMPLATE&';
+                                        $link .= http_build_query([
+                                            'text' =>  $activity->getName(),
+                                            'dates' => $begin . '/' . $end,
+                                            'location' => $activity->getLocation(),
+                                        ]);
+                                    ?>
+                                    
+                                     <a class="btn btn-default" target="_blank" href= "<?php echo $link ?>" class="btn"> Google Calendar </a>
```
Comment (Kevinjil) [inline module/Activity/view/activity/activity/view.phtml]: Duplicate attribute `class` on `<a>` tag.

**Pull request merged**